### PR TITLE
4ddx:Fix/text-color-hover-handbook

### LINKED ISSUE #4706: [Community] Handbook: text color on hover of card - too low contrast
### Current Behavior

Similar to Issue #4615 , the contrast is too low on this page : [handbook-page](https://layer5.io/community/handbook)

### Desired Situation

Make text white on hover.

image

@leecalcote @Ghat0tkach I can fix this issue if any one of you assigns it to me 

  Thanks,
  Best

### PR DESCRIPTION
**Pull request by 4ddx**
Signed-off-by: Ishan Kumar <ishanakshar@gmail.com>
**Description**

This PR fixes #4706 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.
 

fix text-color on hover at [page](https://layer5.io/community/handbook)

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit a99a355115694753fbed9659183a79fbbeadbb84 at: https://64d1449325e28c1ed2e6e3c3--layer5.netlify.app

**Comment by 4ddx** [comment]
@Ghat0tkach @leecalcote any updates on this pr?

**Review by 4ddx — COMMENTED** [review]
Self Reviewed. LGTM.

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
@4ddx, please refer to [contributing.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md), to understand how we reference color schemes.

**Comment by 4ddx** [comment]
@thisiskaransgit white is a defined color in the color scheme ->lighttheme

 #4702  was merged and i had used "white".
Attaching image for your reference:

Thanks,
Best

image

**Review by leecalcote — CHANGES_REQUESTED** [review]
> @4ddx, please refer to [contributing.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md), to understand how we reference color schemes.

@4ddx thanks for this PR. Please heed @thisiskaransgit's counsel. This site's approach to color theming is slightly different.

**Comment by 4ddx** [comment]
@leecalcote I'm not quite sure I understand, I read through the contributing.md, went to themeStyles.js, and white is indeed a defined color

Sorry for the confusion

**Comment by leecalcote** [comment]
Read more closely. https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#adding-dark-mode-for-components

**Comment by 4ddx** [comment]
Hey @leecalcote I think we have a misunderstanding here.

It does not matter whether it's light mode, or dark mode, when hovered over, the text should be white.
This is exactly what issue #4615 was about.

What is it exactly that I'm missing? 

Thanks,
Best

**Comment by leecalcote** [comment]
The difference is sustainability. It's better to have theme-based references; better to use the framework than to make absolute reference.

Example: In a couple of months from now, when the community decides that #FFFFFF (white) for text presented in dark mode, stings the eyes and it should be softened to #ECECECE (off-white). The difference is a one-line change versus crawling through hundreds of code files.

Does this help?

**Comment by l5io** [comment]
🚀 Preview for commit 79eb9992fed8dd653cb605b3f4e6f2ede82fce53 at: https://64d6161ba3736a52863551dc--layer5.netlify.app

**Comment by Ghat0tkach** [comment]
Hii @4ddx 

Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

**Comment by 4ddx** [comment]
Hey @Ghat0tkach 

I'm denied access to the google link you sent, I just requested. Please check?

**Comment by Ghat0tkach** [comment]
> Hey @Ghat0tkach 
> 
> I'm denied access to the google link you sent, I just requested. Please check?

Haven't you filled newcomers form which gives you access to a list of valuable resources? Or joined us on Slack yet?

**Comment by 4ddx** [comment]
> > Hey @Ghat0tkach
> > I'm denied access to the google link you sent, I just requested. Please check?
> 
> Haven't you filled newcomers form which gives you access to a list of valuable resources? Or joined us on Slack yet?

I'm on slack, but haven't filled any newcomers form

**Comment by 4ddx** [comment]
Okay, I just filled out the form as well

File: src/components/HandbookCard/HandbookCard.style.js [hunk 1]
```diff
@@ -99,6 +99,6 @@ export const HandbookCardWrapper = styled.div`
 	}
 
 	.handbook__card:hover .handbook__card--head {
-	  color: ${props => props.theme.text};
+	  color: ${props => props.theme.navigatorColor};
```
Comment (thisiskaransgit) [inline src/components/HandbookCard/HandbookCard.style.js:102]:
```suggestion
	  color: ${props => props.theme.white};
```

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]

File: src/components/HandbookCard/HandbookCard.style.js (hunk 1 above)
Comment (thisiskaransgit) [reply src/components/HandbookCard/HandbookCard.style.js:102]: @4ddx, do this and we're good to go

**Comment by 4ddx** [comment]
Hey, my bad i forgot to sign-off

**Comment by 4ddx** [comment]
Hi @thisiskaransgit  I'm having some troubles with the sign-off

I also opened another pull request, and closed it, sorry for that.

**Comment by l5io** [comment]
🚀 Preview for commit 6ee18b52ab1e5077d823ee21635d3a5b0a733a01 at: https://64e1a901efff1a385b41a831--layer5.netlify.app

**Comment by 4ddx** [comment]
@thisiskaransgit I'm having problems with rebasing.

I'll be closing this pull request and opening a new one. Sorry for the delay.

**Pull request closed without merging**